astro/src/content/docs/get-started/core-concepts/editions-features.mdx

```diff
@@ -31,7 +33,11 @@ Please review the [pricing page](/pricing) for much more information on editions
 
 ## Feature List
 
-Here are the major features available in each paid edition.
+Here are the major features available in each licensed edition.
+
+### Base Licensed Features
```

Comment:
All  licensed editions will have WebAuthn now.

Reply:
did we decide to call it `Base`? I thought we were calling it `community licensed`.